mpags-cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <vector>
+#include <string.h>
+#include <iostream>
+#include <stdio.h>
+
+int main(int argc, char* argv[]) {
+
+/*
+A simple C++ tool to encrypt/decrypt text
+*/
+
+std::string version = "0.1.0";
+
+const std::vector<std::string> cmdLineArgs { argv, argv+argc };
+
+for (int i = 0; i < cmdLineArgs.size(); i++){
+    auto arg = cmdLineArgs[i];
```

Comment:
In the next session we will cover references, which would be helpful here to avoid a copy of the string at this line.
```suggestion
    auto& arg = cmdLineArgs[i];
```